### ISSUE TITLE
Fusion 360 build instructions

### ISSUE THREAD

**Issue by AntonEvmenenko** [JesusFreke/lalboard#9]
Hi @JesusFreke!

I want to thank you for creating this very interesting project. 

But, TBH, for me the most interesting part of this project is the way you used to create 3D models. I know about OpenScad and I like this idea of converting code to 3D models. It's very cool that you used the same idea but for Fusion 360.

But, unfortunately, it's pretty hard to figure out how to get a Fusion360 to "compile" this. Maybe it makes sense to add some instructions about this to Readme? Something like this:

1. Install fscad (see this [hint](https://github.com/JesusFreke/fscad/wiki/Installation) about installation).
2. Copy the content of `parts` folder to `%appdata%\Roaming\Autodesk\Autodesk Fusion 360\API\Scripts` (this path is valid for Windows, for macOS please check [this link](https://knowledge.autodesk.com/support/fusion-360/troubleshooting/caas/sfdcarticles/sfdcarticles/How-to-install-an-ADD-IN-and-Script-in-Fusion-360.html)).
3. Copy `lalboard.py` to `%appdata%\Roaming\Autodesk\Autodesk Fusion 360\API` (the same about the path).
4. Start Fusion 360, go to `Tools->Scripts and Add-ins...` select the script from `My Scripts` folder and press Run.

Also, IMHO it would be very cool to have `message_box_on_error` enabled by default (this would make it easier to solve any problems).

**Comment by JesusFreke** [comment]
I do have a "getting started" guide/walkthrough for fscad at https://github.com/JesusFreke/fscad/wiki/Getting-started-with-Intellij-IDEA-(Windows). But yeah, it would probably be a good idea to at least have a pointer to that here or something. I can add a "Building" section to the readme.

**Comment by AntonEvmenenko** [comment]
> it would probably be a good idea to at least have a pointer to that here or something

Exactly! It's cool that this guide exists, but I couldn't find it when I was trying to figure out how to build your project. This may mean that some random guys like me won't be able to find it either :) So, I guess, it's worth adding a little clarity.
